assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -155,6 +166,35 @@ const baseActions = {
 			}
 
 			yield propertyActions.selectProperty( uaProperty?.id, uaProperty?.internalWebPropertyId ); // eslint-disable-line sitekit/acronym-case
+
+			if ( ! isFeatureEnabled( 'ga4setup' ) ) {
+				return;
+			}
+
+			registry.dispatch( STORE_NAME ).setPrimaryPropertyType( PROPERTY_TYPE_UA );
+
+			yield Data.commonActions.await( registry.dispatch( MODULES_ANALYTICS_4 ).waitForProperties( accountID ) );
+
+			const ga4Properties = registry.select( MODULES_ANALYTICS_4 ).getProperties( accountID );
+			const ga4Property = yield Data.commonActions.await(
+				registry.dispatch( MODULES_ANALYTICS_4 ).matchPropertyByURL(
+					ga4Properties.map( ( property ) => property._id ),
+					registry.select( CORE_SITE ).getReferenceSiteURL(),
+				)
+			);
+
+			yield Data.commonActions.await(
+				registry.dispatch( MODULES_ANALYTICS_4 ).selectProperty( ga4Property?._id || GA4_PROPERTY_CREATE ),
+			);
```

Comment:
As in the previous PR, please update these actions to `yield {actionCreator}()` rather than awaiting registry.dispatch.

Reply:
@aaemnnosttv this action is from the `analytics-4` module, so if i call it directly it will be applied to the `analytics` datastore, right?

Reply:
Yes, sorry I missed that. For some reason I was thinking this was in the GA4 store – nevermind! 😄